src/renderer/components/Index/Inventory.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="pug">
+.backgrounds_container
+    button.hover-button
+        p Inventory
+    - for (let a=0; a < 10; a++)
+        .invetory-item
+</template>
+
+<style lang="stylus">
+
+$color-1 = #12151a
```

Comment:
Don't write variables like that in local css. Move them to layout.vue or something like that. Variables should be declared in top level block.